hooks/post_gen_project.py

```diff
@@ -189,24 +195,29 @@ def update_local_workflows(central_workflows):
 
         for name in local_workflows - central_workflow_names:
             (LOCAL_WORKFLOW_DIR / name).unlink()
-            print(f"Removed workflow {name}")
-
 
     try:
         LOCAL_WORKFLOW_DIR.mkdir(parents=True, exist_ok=True)
         central_workflows = get_central_workflows()
         update_local_workflows(central_workflows)
-        print("Workflow synchronization completed successfully")
     except Exception as e:
         print(f"Error: {str(e)}")
 
 
 def main():
+    """Execute when user runs cookiecutter."""
     if "." in "{{ cookiecutter.project_name }}":
         add_supermodules(ROOT, "{{ cookiecutter.project_name }}")
-    if "{{ cookiecutter.have_c_code }}" == "y":
+    if "{{ cookiecutter.project_needs_c_code_compiled }}" == "Yes":
         wrapper_setup()
     update_workflow()
+    print(
```

Comment:
Once the user prompt is done, I provide (1) immediate next step by `cd` into the project directory (2) how to find more info (3) how to receive help